grc/tests/test_generator.py

```diff
@@ -17,12 +17,15 @@
 # the Free Software Foundation, Inc., 51 Franklin Street,
 # Boston, MA 02110-1301, USA.
 
+import pytest
```

Comment:
Does `grc/CMakeLists.txt` already check for `pytest`? If not, please add that check with respect to the `ENABLE_TESTS` just in GRC.

Reply:
Yes, but the check is in `grc/tests/CMakeLists.txt`

Reply:
:+1: